docs/custom-flows/multi-session-applications.mdx

```diff
@@ -1,5 +1,183 @@
 ---
-sanity_slug: /docs/authentication/multi-session-applications
+title: Multi-session applications
+description: Learn how to configure and work with multi-session applications
 ---
 
-# This is a stub
+# Multi-session applications
+
+One of the most powerful features that Clerk provides out of the box is multi-session applications.
+
+A multi-session application is an application that allows multiple accounts to be signed-in from the same browser at the same time. The user can switch from one account to another seamlessly. Each account is independent from the rest and has access to different resources. It's most clearly explained by the default [`<UserButton />`][user-button-ref] below.
+
+<Images
+  width={880}
```

Comment:
Image load is now fixed but it looked way too wide to me - this felt a bit more reasonable

```suggestion
  width={400}
```